company.py

```diff
@@ -17,3 +18,19 @@ class Company:
 
     header_html = fields.Text('Header Html')
     footer_html = fields.Text('Footer Html')
+
+    def render_html(self, position):
+        """
+        Render HTML for the position specified
+
+        :position: 'header' or 'footer'
+        """
+        assert position in ['header', 'footer']
+
+        if position == 'header':
+            html_content = self.header_html
+        else:
+            html_content = self.footer_html
+
+        env = ReportWebkit.get_environment()
```

Comment:
This is not the right approach. This will still create the environment again and again.